styles/main/main.scss

```diff
@@ -60,9 +60,6 @@ body {
 	-webkit-font-smoothing: antialiased;
 	-moz-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
-	&.view {
-		background-color: #0f0f0f;
```

Comment:
Oh, I see now where `0f` came from... Still, I don't like the rare `view` mode taking precedence over a lot more common regular photo view.

Reply:
Right, this is the source of that background color. In order to keep the discussion at one place, I will close this conversation here. We should continue the discussion in  https://github.com/LycheeOrg/Lychee-front/pull/323#discussion_r1001350913.